.github/workflows/ci.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
         -
             uses: actions/setup-python@v1
             with:
-                python-version: '3.x'
+                python-version: '3.8'
```

Comment:
We should do a matrix job here for 3.6...3.8.

Reply:
Good idea, added.